WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -62,30 +62,24 @@ public async Task<PrivacySuggestionsResult> BuildPrivacySuggestionsAsync(Transac
 
 		using (await _asyncLock.LockAsync(CancellationToken.None))
 		{
-			try
+			result.Add(VerifyLabels(info, transactionResult));
+			result.Add(VerifyPrivacyLevel(info, transactionResult));
+			result.Add(VerifyConsolidation(transactionResult));
+			result.Add(VerifyUnconfirmedInputs(transactionResult));
+			result.Add(VerifyCoinjoiningInputs(transactionResult));
```

Comment:
Note for the future:
if one of these is a heavy operation, it will block all the other results.
This is not the case at the moment.